maven-model/pom.xml

```diff
@@ -37,16 +37,22 @@ under the License.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-api-model</artifactId>
-      <version>4.0.0-alpha-3-SNAPSHOT</version>
+      <version>${project.version}</version>
```

Comment:
Actually, this should be better added to dependency management of the parent pom along with all other modules defined in the build.

Reply:
Will do. I'll get on it this weekend .